src/azure-cli/requirements.py3.windows.txt

```diff
@@ -125,6 +125,7 @@ PyGithub==1.55
 PyJWT==2.4.0
 PyMySQL==1.0.2
 PyNaCl==1.5.0
+pyodbc==4.0.35
```

Comment:
It seems more dependencies are included. What's `pyodbc` used to?

Reply:
BTW, I highly recommended move `Service Connector` into azure-cli-extension because it involves lots of third parts libraries. And it may block other modules or extensions like `az interactive` [issue](https://github.com/Azure/azure-cli/issues/24213)

Reply:
It's a valid concern, we will take it into serious consideration and discuss it with the team. While before that, how about we let the new command group in the core to keep consistency.

For the library, if it indeed has some conflicts with other components, i think we can discuss if there are any work arounds there

Reply:
@bebound is evaluating the `pyodbc` package.

Reply:
Thanks @kairu-ms  and @bebound  As we discussed, `pyodbc` doesn't support well on linux and it does fail many tests on the pipeline  https://github.com/Azure/azure-cli/pull/24905/checks?check_run_id=10246207156 
The homebrew test also failed because it needs to install unixodbc fisrt. The library home page shows the same requirement: https://github.com/mkleehammer/pyodbc
It also say `On Windows, the ODBC driver manager is built-in`, which means Windows doesn't need prerequisite when installing pyodbc.
So I remove pyodbc from linux and darwin requirement file and keep the windows requirement file.